app/assets/stylesheets/ghetto_styles.css.scss

```diff
@@ -0,0 +1,1500 @@
+body.idle #post-nav .nav-arrow.right {
+  margin-right: -40px;
+}
+/* line 17, ../../app/assets/stylesheets/new_styles/_viewer_nav.scss */
+body.idle #post-nav .nav-arrow.left {
+  margin-left: -40px;
+}
+
+/* line 25, ../../app/assets/stylesheets/new_styles/_viewer_nav.scss */
+.nav-arrow {
+  filter: alpha(opacity=80);
+  -moz-opacity: 0.8;
+  -khtml-opacity: 0.8;
+  opacity: 0.8;
+  -webkit-transition: all 0.3s linear;
+  -moz-transition: all 0.3s linear;
+  -o-transition: all 0.3s linear;
+  transition: all 0.3s linear;
+  display: table;
+  position: fixed;
+  height: 100%;
+  z-index: 3;
+  top: 0;
+  padding: 0 1.2%;
+  background-color: rgba(0, 0, 0, 0);
+}
+/* line 39, ../../app/assets/stylesheets/new_styles/_viewer_nav.scss */
+.nav-arrow .nav-arrow-inner {
+  padding: 0;
+  margin: 0;
+  display: table-cell;
+  vertical-align: middle;
+}
+/* line 45, ../../app/assets/stylesheets/new_styles/_viewer_nav.scss */
+.nav-arrow img {
+  filter: alpha(opacity=50);
+  -moz-opacity: 0.5;
+  -khtml-opacity: 0.5;
+  opacity: 0.5;
+  height: 30px;
+  width: 30px;
+  /* half the size of the nav arrows on mobile phones */
+}
+@media (max-width: 480px) {
+  /* line 45, ../../app/assets/stylesheets/new_styles/_viewer_nav.scss */
+  .nav-arrow img {
+    height: 15px;
+    width: 15px;
+  }
+}
+/* line 58, ../../app/assets/stylesheets/new_styles/_viewer_nav.scss */
+.nav-arrow.left {
+  left: 0;
+  padding-right: 19px;
+}
+/* line 63, ../../app/assets/stylesheets/new_styles/_viewer_nav.scss */
+.nav-arrow.right {
+  right: 0;
+  padding-left: 19px;
+}
+/* line 68, ../../app/assets/stylesheets/new_styles/_viewer_nav.scss */
+.nav-arrow:hover {
+  background-color: rgba(0, 0, 0, 0.1);
+}
+
+/* font overrides */
+/* Roboto */
+@font-face {
+  font-family: Roboto;
+  src: url(/assets/fonts/Roboto-Regular.ttf);
+  weight: normal;
+}
+
+@font-face {
+  font-family: Roboto-Bold;
+  src: url(/assets/fonts/Roboto-Bold.ttf);
+  weight: normal;
+}
+
+@font-face {
+  font-family: Roboto-Light;
+  src: url(/assets/fonts/Roboto-Light.ttf);
+  weight: normal;
+}
+
+/* line 20, ../../app/assets/stylesheets/new_styles/_typography.scss */
+body, p, h1, h2, h3, h4, h5, h6, textarea, input, * {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: normal;
+}
+
+/* line 25, ../../app/assets/stylesheets/new_styles/_typography.scss */
+a {
+  font-family: inherit;
+}
+
+/* login */
+/* line 1, ../../app/assets/stylesheets/new_styles/_login.scss */
+#login {
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding-top: 200px;
+  min-width: 100%;
+  text-align: center;
+}
+/* line 12, ../../app/assets/stylesheets/new_styles/_login.scss */
+#login input[type=text],
+#login input[type=password] {
+  width: 120px;
+}
+/* line 16, ../../app/assets/stylesheets/new_styles/_login.scss */
+#login #huge-text {
+  font-family: Roboto-Light;
+  font-size: 200px;
+  color: #ddd;
+  text-shadow: 0 1px 0 white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.